gem/pupperware.gemspec

```diff
@@ -32,9 +32,9 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", "~> 1.16"
-  spec.add_development_dependency "rake", "~> 10.0"
+  # spec.add_development_dependency "bundler", "~> 1.16"
+  # spec.add_development_dependency "rake", "~> 10.0"
 
-  spec.add_runtime_dependency "rspec", "~> 3.0"
-  spec.add_runtime_dependency "json"
+  # spec.add_runtime_dependency "rspec", "~> 3.0"
+  # spec.add_runtime_dependency "json"
```

Comment:
Do we need any of these dependencies?

**Removing them makes the bundle install of our shared gem faster!**

I think these dependencies came from bundler generating the initial gem for us.
But we don't use any of it I don't think; maybe we would be if we were publishing the gem on RubyGems?

Reply:
This change, along with grouping our dependencies in the Gemfile will speed up the bundle calls and reduce the slowdown we experience during each make invocation.

Grouping dependencies: https://github.com/puppetlabs/pe-orchestration-services/pull/162/commits/b9863337dbba7199b34044981450735a28a5efbb

Reply:
Remove all this cruft here: https://github.com/puppetlabs/pupperware/pull/170